.github/CODE_OF_CONDUCT_EN.md

```diff
@@ -2,19 +2,19 @@ version 1.0, 2020/3/3
 # Our mission
 
 * We will work to protect citizen's life and health.
```

Comment:
citizen's --> singular
citizens' --> plural

Maybe something like "We will work to protect the life and health of the citizens of Tokyo / Japan"

Reply:
The Japanese version says 都民、so I guess it'll be "the citizens of Tokyo"

Reply:
Sorry, I was referring to the usage of "citizen" in the section titled "Build with people".
>We will welcome any citizens' contributions

It sounded a bit strange to me, since it seems contributions from *anyone* are welcome.

Reply:
I like your suggested change here, though! :+1: